packages/utils/odsp-doclib-utils/package.json

```diff
@@ -13,6 +13,36 @@
 	"sideEffects": false,
 	"exports": {
```

Comment:
Here is the real change:

"." -> `@public`
"./beta" -> `@beta`
"./fruit" -> `@alpha`
"./internal" -> `@internal`